vnext/template/cpp-app/src/pch.h

```diff
@@ -6,16 +6,12 @@
 #include <restrictederrorinfo.h>
 #include <unknwn.h>
 #include <windows.h>
+#include <CppWinRTIncludes.h>
```

Comment:
This seems like it would balloon template PCH. Can we avoid this? Seems likely to cause customers worse issues for compiler OOM, potentially perf.

Reply:
I think this is smaller than it used to be though... Need to be careful here though.